types/pnpapi/pnpapi-tests.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="node"/>
```

Comment:
I’ve added `@types/node` to `devDependencies` so that changes to **Node** type definitions won’t cause breakage like with `@types/webpack‑env`: <https://github.com/DefinitelyTyped/DefinitelyTyped/issues/47602> (PR <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47516>).